schema/deploy/computed_columns/project_revision_project_manager_form_changes_by_label.sql

```diff
@@ -9,26 +9,14 @@ returns setof cif.manager_form_changes_by_label_composite_return
 as
 $computed_column$
 
-  with project_form_change_history as (
-    select fc.*
-      from cif.form_change fc
-      join cif.project_manager_label pml on cast(new_form_data->>'projectManagerLabelId' as integer) = pml.id
-        and (fc.updated_at, fc.id) = (select max(updated_at), max(id) from cif.form_change where cast(new_form_data->>'projectManagerLabelId' as integer) = pml.id)
-      where project_revision_id = $1.id
-        and form_data_schema_name='cif'
-        and form_data_table_name='project_manager'
-    union
-    select fc.*
-      from cif.form_change fc
-      join cif.project_manager pm on
-        fc.form_data_record_id = pm.id
-        and pm.project_id = $1.project_id
-        and (fc.updated_at, fc.id) = (select max(updated_at), max(id) from cif.form_change where form_data_record_id = pm.id)
-  )
-  select label, row(project_form_change_history.*) as form_change
-    from project_form_change_history
+  with latest_changes as (
+    select distinct on (new_form_data->'projectManagerLabelId') * from cif.form_change
```

Comment:
The same `projectManagerLabelId` shouldn't appear more than once for a given `project_revision_id`, so the `distinct on` isn't needed.

Reply:
unless 2 users are concurrently updating the same revision?

Reply:
If 2 users were able to do concurrent updates, they would still be updating the same `form_change` record

Reply:
ah indeed!